### DISCUSSION TITLE
Running MODE with all initializations and leads in one file

### DISCUSSION THREAD

**Discussion by cohenjt** [dtcenter/METplus#2467] [Use Cases] [answered]
I am interested in applying MODE to a large dataset with multiple ensemble members, initialization times, and lead times, and was wondering what the best workflow would be. I have successfully applied MODE to a single timestep for the forecast and observed fields, but now would like to scale that up. From reading the documentation and discussions, it looks like I should use python embedding and the MODE wrapper to read in separate files for each member, init, and lead. Is this the case, or is there a way to pass the entire forecast field as a single file and compare every combination of forecast/observation? I just wanted to check if that option was available before splitting the forecast file into multiple files for each possible timestep. I can provide more specific details and examples if necessary. Thank you!

**Comment by cohenjt** [comment, marked as answer]
This use case helped me solve this: https://github.com/dtcenter/METplus/blob/main_v5.1/parm/use_cases/model_applications/clouds/GridStat_fcstGFS_obsERA5_lowAndTotalCloudFrac.conf

**Reply by jprestop** [reply]
@cohenjt Apologies for the lack of replies here. Somehow this fell through and did not receive a reply.  It's great to know that you found a solution.  I'll go ahead and mark this as answered and lock it.  Please feel free to post a new Discussion with any other questions you have as you work with METplus.
